search-service/src/main/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeService.kt

```diff
@@ -307,6 +307,31 @@ class TemporalEntityAttributeService(
             entity[it.key] = it.value
         }
 
+        emptyTemporalAttributesArrays(entity, withTemporalValues)
         return JsonLdEntity(entity, jsonLdEntity.contexts)
     }
+
+    private fun emptyTemporalAttributesArrays(entity: MutableMap<String, Any>, withTemporalValues: Boolean) {
```

Comment:
I think we should find a better name for this function.

Reply:
What about emptyTemporalAttritbutesHavingValueField ? (empty is a verb here)